VideoAPI/VideoApi.Contract/Responses/CvpAudioRecordingResponse.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections.Generic;
+
+namespace VideoApi.Contract.Responses
+{
+    public class CvpAudioRecordingResponse
+    {
+        public IEnumerable<CvpAudioFile> Results { get; set; }    
+    }
```

Comment:
Why we cannot return just IEnumerable<CvpAudioFile> and not create additional class?